### ISSUE TITLE
Duplicate export error for CommonJS

### ISSUE THREAD

**Issue by guybedford** [rollup/plugins#652]
- Rollup Plugin Name: plugin-commonjs
- Rollup Plugin Version: 16.0.0
- Rollup Version: 2.33.22

Example module:

dep.cjs
```js
exports.init = void 0;
function init () {
}
exports.init = init;
```

main.mjs
```js
import test from './test.cjs';
test.init();
```

Built with:

```
rollup --plugin @rollup/plugin-commonjs main.mjs
```

### Expected Behavior

Should build correctly.

### Actual Behavior

RollupJS reports the following error:

```js
node_modules/es-module-lexer/dist/lexer.cjs (13:19)
11: export { parse_1 as parse };
12: export { init_1 as init };
13: export { init_2 as init };
                       ^
Error: Duplicate export 'init' (Note that you need plugins to import files that are not JavaScript)
```

**Comment by lukastaegert** [comment]
Fixed in #537

**Comment by guybedford** [comment]
Great, let's get that deployed please :)
